source/style-guide/markup/directives/code-blocks.txt

```diff
@@ -2,20 +2,444 @@
+      :emphasize-lines: <line numbers>
+      :caption: <caption>
+      :dedent: <number>
+
+Options
+~~~~~~~
+
+.. list-table::
+
+   * - ``<file path>``
+     - **Required.** The path to the file in the repository that contains the code you
+       want to show.
+
+       **Default:** None
+
+   * - ``:language: <language>``
+     - The language to use for syntax highlighting. 
+       
+       For a complete list of supported languages, see the
+       `leafygreen-ui GitHub repository. <https://github.com/mongodb/leafygreen-ui/tree/main/packages/code#properties>`__
+
+       **Default:** None
+ 
+   * - ``:copyable: <boolean>``
+     - Whether to show the Copy button for the code sample.
+
+       **Default:** ``true``
+
+   * - ``:start-after: <line of code>``
+     - The line of code before the first line that you want to include in the sample.
+
+       **Default:** Beginning of file
+   
+   * - ``:end-before: <line of code>``
+     - The line of code after the last line that you want to include in the sample.
+
+       **Default:** End of file
+
+   * - ``:linenos:``
+     - Shows line numbers for the code sample.
+
+       **Default:** None
+ 
+   * - ``:lineno-start:``
+     - The number at which to start line numbering. This option requires the
+       ``:linenos:`` option.
+
+       **Default:** 1
+
+   * - ``:emphasize-lines: <line number>``
+     - Lines to highlight for the code sample. You can specify a range 
+       of line numbers, individual line numbers, or both.
+
+       **Default:** None
+ 
+   * - ``:caption: <caption text>``
+     - The caption for the code sample.
+
+       **Default:** None
+
+   * - ``:dedent: <number of spaces>``
+     - The number of spaces to remove from the beginning of each line of code.
+       Use this option if the code you want to show is indented in the code file.
+
+       **Default:** None 
+
+Example 1
+~~~~~~~~~
+
+Suppose you want to include samples from the following Python file in a docs page:
+
+.. literalinclude:: /includes/python-sample.py
+   :language: python
+
+The following ``literalinclude`` directive starts with the first line of the file and
+ends before the line that reads ``print("...but this line won't.")``: 
+
+.. code-block::
+ 
+   .. literalinclude:: /includes/python-sample.py
+      :copyable: false
+      :end-before: print("...but this line won't.")
+      :caption: Sample Python code from another file
+  
+The previous ``literalinclude`` directive creates the following output:
+
+.. literalinclude:: /includes/python-sample.py
+   :copyable: false
+   :end-before: print("...but this line won't.")
+   :caption: Sample Python code from another file
+
+Example 2
+~~~~~~~~~
+
+You can use comments to mark the points where the ``literalinclude`` directive
+starts and finishes pulling in code.
+By using comments instead of code, you can unambiguously specify the start and end of
+the code sample, regardless of how many samples you're pulling in from the file.
```

Comment:
updated

Reply:
I wasn't able to locate this info other than the start-after and end-before list table descriptions. I still think it would be helpful to describe this in the description under the Literal Includes header, but happy to let you decide what you think would work best.